js-objects/objects.js

```diff
@@ -0,0 +1,138 @@
+//Task №1 
+console.log("Task №1");
+
+const automationQA = {};
+
+automationQA.programmerName = "John";
+automationQA.programmingLanguages = "JavaScript";
+automationQA.age = 27;
+automationQA.languages = ["Ukrainian", "English"];
+automationQA.preferences = { 
+    basicPreferences: "table" 
+};
+
+console.log(automationQA);
+
+
+//Task №2 
+console.log("Task №2");
+
+const room = {
+    name: "Living Room",
+    size: 12,
+    window: true
+};
+
+console.log("Room: " + room.name); 
+console.log("Window: " + room.window); 
+
+//Task №3 
+console.log("Task №3");
+
+const movie = {
+    title: "Interstellar",
+    releaseYear: 2014,
+    genre: "Science fiction"
+};
+
+movie.genre = "Action thriller";
+movie.title = "John Wick";
+
+movie.actors = { mainRole: "Keanu Reeves", secondRole: "Michael Nyqvist" };
+movie.director = "	Chad Stahelski";
```

Comment:
It is necessary to have empty space here?
```suggestion
movie.director = "Chad Stahelski";
```